CHANGELOG-2.1.md

```diff
@@ -1,3 +1,21 @@
+# Release notes for v2.1.5 (Changelog since v2.1.4)
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Cherry pick of #413: Bug fix to allow creation of snapshot content if pvc finalizer exists, even if pvc is marked for deletion.
+- Cherry pick of #478: Call dynamic VS content creation unconditionally.
```

Comment:
Did you cherry pick #478 in this PR?  I don't see it.

Reply:
the wording is a bit confusing. But #478 itself is the actual cherrypick to release-1.2

Reply:
Can you reword these two lines following this changelog?
https://github.com/kubernetes-csi/external-snapshotter/blob/v3.0.3/CHANGELOG/CHANGELOG-3.0.md#bug-or-regression

Reply:
Sorry for the confusion, done.

Note that #413 is cherrypicked in this PR (#488), I hope that's ok.